README.md

```diff
@@ -68,12 +68,49 @@ docker compose --profile all down --volumes
 
 ## Advanced 👷
 
+### Development
 If you are a developer on `bee-api` or `bee-ui` and want to run only the supporting infrastructure,
 use the profile `infra`, e.g.:
 
 ```shell
 docker compose --profile infra up -d
 ```
+### podman compose
+
+Podman supports multiple [providers](https://docs.podman.io/en/latest/markdown/podman-compose.1.html), specifically `docker-compose` and `podman-compose`. 
+
+For the bee framework the `docker-compose` provider is required. This will need to be installed`
+
+### Connecting to host resources
+
+There are many networking options in both [Podman](https://github.com/containers/podman/blob/main/docs/tutorials/basic_networking.md) and [Docker](https://docs.docker.com/engine/network/)
+
+The bee stack does not explicitly configure a network, so by default typically a bridge network is generated automatically. Additionally a specific host name is made available to the container which represents the host. This can be used to access other services on that host, such as ollama running locally.
+
+The hostnames are:
+* host.containers.internal (podman)
+* host.docker.internal (docker)
```

Comment:
As author I'll add a note that it seems recent versions of podman do also support the docker host ( https://github.com/containers/podman/issues/19361 )but I've left the text as-is for now